diff --git a/node_modules/react-native-pell-rich-editor/src/editor.js b/node_modules/react-native-pell-rich-editor/src/editor.js
index 6559109..fcf2405 100644
--- a/node_modules/react-native-pell-rich-editor/src/editor.js
+++ b/node_modules/react-native-pell-rich-editor/src/editor.js
@@ -11,9 +11,43 @@ function getContentCSS() {
         .x-todo-box {position: relative; left: -24px;}
         .x-todo-box input{position: absolute;}
         blockquote{border-left: 6px solid #ddd;padding: 5px 0 5px 10px;margin: 15px 0 15px 15px;}
-        hr{display: block;height: 0; border: 0;border-top: 1px solid #ccc; margin: 15px 0; padding: 0;}
-        pre{padding: 10px 5px 10px 10px;margin: 15px 0;display: block;line-height: 18px;background: #F0F0F0;border-radius: 6px;font-size: 13px; font-family: 'monaco', 'Consolas', "Liberation Mono", Courier, monospace; word-break: break-all; word-wrap: break-word;overflow-x: auto;}
-        pre code {display: block;font-size: inherit;white-space: pre-wrap;color: inherit;}
+        hr{display: block;height: 0; border: 0;border-top: 3px solid #FACCD6; margin: 15px 0; padding: 0;}
+        pre{
+            padding: 10px 5px 10px 10px;
+            margin: 15px 0;
+            display: block;
+            line-height: 18px;
+            border-radius: 6px;
+            font-size: 18px; 
+            word-break: break-all; 
+            word-wrap: break-word;
+            overflow-x: auto;
+            font-family: ancizar;
+            position: relative;
+            box-shadow: 0 8px 20px rgba(0,0,0,.08);
+            border: 2px solid #FACCD6;
+            background-color: #fff;
+        }
+        pre img {
+            
+            overflow: hidden;
+        }
+        pre::after {
+            content: "";
+            display: inline-block;
+            background-image: url("https://mollydigital.manu-scholz.com/wp-content/uploads/2025/08/note-decoration.png");
+            background-size: contain;
+            background-repeat: no-repeat;
+            width: 72px;
+            height: 72px;
+            transform: rotate(36deg);
+            position: absolute;
+            top: 0px;
+            right: 0px;
+            overflow: hidden;
+
+}
+        pre code {display: block;font-size: inherit;white-space: pre-wrap;color: inherit;font-family: ancizar}
     </style>
     `;
 }
